ydb/docs/en/core/reference/ydb-sdk/coordination.md

```diff
@@ -0,0 +1,330 @@
+# Working with coordination nodes
+
+This article describes how to use the {{ ydb-short-name }} SDK to coordinate the work of multiple client application instances using [coordination nodes](../../concepts/datamodel/coordination-node.md) and their semaphores.
+
+## Creating a coordination node
+
+Coordination nodes are created in {{ ydb-short-name }} databases in the same namespace as other schema objects, such as [tables](../../concepts/datamodel/table.md) and [topics](../../concepts/topic.md).
+
+{% list tabs %}
+
+- Go
+
+    {% list tabs group=lang %}
+
+    - Go
+
+        ```go
+        err := db.Coordination().CreateNode(ctx,
+            "/path/to/mynode",
+        )
+        ```
+
+    {% endlist %}
```

Comment:
```suggestion
    ```go
    err := db.Coordination().CreateNode(ctx,
        "/path/to/mynode",
    )
    ```
```